Description: Make RenderingContext in sitetools extend RenderingContext in doxia
 to preserve the backward compatibility. This patch can be removed once no code
 in Debian imports org.apache.maven.doxia.sink.render.RenderingContext.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/RenderingContext.java
+++ b/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/RenderingContext.java
@@ -33,7 +33,7 @@
  * @version $Id: RenderingContext.java 1720929 2015-12-19 14:09:41Z hboutemy $
  * @since 1.5 (was since 1.1 in o.a.m.d.sink.render)
  */
-public class RenderingContext
+public class RenderingContext extends org.apache.maven.doxia.sink.render.RenderingContext
 {
     private final File basedir;
 
@@ -76,6 +76,7 @@
      */
     public RenderingContext( File basedir, String document, String parserId, String extension )
     {
+        super( basedir, document, parserId, extension );
         this.basedir = basedir;
         this.extension = extension;
         this.inputName = document;
--- a/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/sink/SiteRendererSink.java
+++ b/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/sink/SiteRendererSink.java
@@ -31,7 +31,7 @@
 import org.apache.maven.doxia.module.xhtml.XhtmlSink;
 import org.apache.maven.doxia.sink.Sink;
 import org.apache.maven.doxia.sink.SinkEventAttributes;
-import org.apache.maven.doxia.siterenderer.RenderingContext;
+import org.apache.maven.doxia.sink.render.RenderingContext;
 import org.apache.maven.doxia.util.HtmlTools;
 import org.codehaus.plexus.util.StringUtils;
 
--- a/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java
+++ b/doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java
@@ -524,7 +524,7 @@
      * @param siteRenderingContext the site rendering context
      * @return
      */
-    protected Context createDocumentVelocityContext( RenderingContext renderingContext,
+    protected Context createDocumentVelocityContext( org.apache.maven.doxia.sink.render.RenderingContext renderingContext,
                                                      SiteRenderingContext siteRenderingContext )
     {
         Context context = createToolManagedVelocityContext( siteRenderingContext );
